packages/components/button/src/IconButton/README.mdx

```diff
@@ -13,9 +13,15 @@ import { PreviewIcon } from '@contentful/f36-icons';
 
 ## Table of contents
 
+- [How to use IconButton](#how-to-use-iconbutton)
 - [Code examples](#code-examples)
 - [Props](#props)
 
+## How to use IconButton
+
+IconButton composes the Button component except that it renders only an icon.  
+You must pass the Icon component you want to render to the `icon` prop, you may also provide the variant and size to the Icon.
```

Comment:
I don't think you can provide size, since it will be overwritten in the `Button`, here https://github.com/contentful/forma-36/blob/eee20b548a1922dfc189eb3636af58076ff5c050/packages/components/button/src/Button.tsx#L60